lib/core/extensions/date_time_extension.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:live_score/core/extensions/localization/app_localizations_context.dart';
+
+extension DateTimeExtension on DateTime {
+  String getFormattedDay(DateTime date, BuildContext context) {
```

Comment:
When creating an extension function, you should use `this` to access a dateTime object.

```
  String getFormattedDay(BuildContext context) {
  DateFormat dayFormat = DateFormat('E');
    DateTime today = DateTime.now();
    String day = date.year == today.year &&
            date.month == today.month &&
            date.day == today.day
        ? context.localizations.today
        : dayFormat.format(this);
    return day;
  }
```

Reply:
This applies for all functions below